Assets/Game/Scripts/Interactions/OutlineOnClickExample.cs

```diff
@@ -0,0 +1,32 @@
+using UnityEngine;
+using ChrisNolet.QuickOutline;
+
+#nullable enable
+
+namespace GameJammers.GGJ2025.GodMode
+{
+    public class OutlineOnClickExample : InteractableObjectBase
```

Comment:
Let's pull the parent Interactions folder into GodMode and pull the two interaction scripts into the folder for organization.